packages/react/src/Logo/Logo.tsx

```diff
@@ -26,16 +26,16 @@ export type LogoBrand =
   | 'vga-verzekeringen'
 
 export type LogoProps = {
-  /** The name of the brand for which to display the logo. */
-  brand?: LogoBrand
+  /** The name of the brand for which to display the logo, or configuration for a custom logo. */
+  brand?: LogoBrand | LogoConfig
```

Comment:
It feels like we're mixing up 2 very different things here. A brand is now either a brand name, or a custom logo. Shouldn't we add a separate prop for that? Something like `customLogo`?